gen.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import shutil
+import subprocess
+
+from jinja2 import Environment, FileSystemLoader
+
+SRC_PATH = 'src'
+WORKPLACE_PATH = 'workplace'
+SCHEMA_FILENAME = 'events.json'
+GEN_PY_FILENAME = 'events.py'
+TEMPLATE_PATH = 'templates'
+INIT_PY_TEMPLATE = 'init_py.template'
+PACKAGE_ROOT = 'google/events'
+
+jinja2_env = Environment(loader=FileSystemLoader(searchpath=TEMPLATE_PATH))
+
+def clean_up():
+    """Clean up leftover artifacts (if any) from previous runs.
+    """
+    try:
+        shutil.rmtree(SRC_PATH)
+    except FileNotFoundError:
+        pass
+
+def run():
+    """Generate the Google Events Library for Python using quicktype.
+    """
+    all_events = []
+    paths = os.walk(WORKPLACE_PATH)
+    for i in paths:
+        path = i[0]
+        subdirectories = i[1]
+        files = i[2]
+        if SCHEMA_FILENAME in files:
+            # Run the quicktype generator
+            args = [
+                'quicktype',
+                '--src',
+                '{}/{}'.format(path, SCHEMA_FILENAME),
+                '--src-lang',
+                'schema',
+                '--lang',
+                'python',
+                '--python-version',
+                '3.7'
```

Comment:
Can we combine flags and flag values?

like 
```py
[
    '--lang python',
    '--python-versoin 3.7'
]
```

Reply:
I am afraid this is not possible. The Python `subprocess` module (and virtually all similar modules in other languages) pass arguments directly to the system call without interpreting them in the same way as shells do.